### PR TITLE
Tweak: Change default Google fonts loading method to "Swap" [ED-7896]

### PR DESCRIPTION
**Pull request by maork-elementor**

### CODE REVIEW

**Review by arielk — CHANGES_REQUESTED** [review]
@maork-elementor please change the new default **only** for **new** installs, not for all sites.

**Comment by maork-elementor** [comment]
> @maork-elementor please change the new default **only** for **new** installs, not for all sites.
done

File: includes/fonts.php [hunk 1]
```diff
@@ -1550,6 +1552,9 @@ public static function get_fonts_by_groups( $groups = [] ) {
 	}
 
 	public static function get_font_display_setting() {
-		return get_option( 'elementor_font_display', 'auto' );
+		// To prevent regressions for old installs, we only apply the new font-display setting for new installs (after 3.8.0).
+		$latest_install = key( Upgrade_Manager::get_installs_history() );
+		$is_new_install = ( ! empty( $latest_install ) ) && version_compare( $latest_install, '3.8.0', '>=' );
+		return get_option( 'elementor_font_display', $is_new_install ? 'swap' : 'auto' );
```
Comment (matipojo) [inline includes/fonts.php:1558]: You don't need to compare the version, the code will be merged to 3.8.0

**Review by matipojo — CHANGES_REQUESTED** [review]
Please cover with tests

File: includes/maintenance.php [hunk 2]
```diff
@@ -45,10 +45,24 @@ public static function activation( $network_wide ) {
...
 	}
 
+	public static function insert_defaults_options() {
+
+		$default_options = [
+			'elementor_font_display' => 'swap',
+		];
+
+		foreach ( $default_options as $option_name => $option_value ) {
+			if ( ! get_option( $option_name ) ) {
```
Comment (matipojo) [inline includes/maintenance.php:60]: Validate it's not 0, `\Elementor\Utils::is_empty`

File: includes/maintenance.php [hunk 3]
```diff
@@ -45,10 +45,24 @@ public static function activation( $network_wide ) {
...
+				update_option( $option_name, $option_value );
```
Comment (matipojo) [inline includes/maintenance.php:61]:
```suggestion
				add_option( $option_name, $option_value );
```

**Review by matipojo — CHANGES_REQUESTED** [review]
Please cover with tests

File: includes/maintenance.php [hunk 4]
```diff
@@ -45,10 +45,24 @@ public static function activation( $network_wide ) {
...
 
 		set_transient( 'elementor_activation_redirect', true, MINUTE_IN_SECONDS );
 	}
 
+	public static function insert_defaults_options() {
+
+		$default_options = [
+			'elementor_font_display' => 'swap',
+		];
+
```
Comment (matipojo) [inline includes/maintenance.php:58]: Only for new installs

**Review by matipojo — CHANGES_REQUESTED** [review]

**Comment by maork-elementor** [comment]
@matipojo 
Test added.

File: includes/settings/settings.php [hunk 5]
```diff
@@ -338,7 +338,7 @@ protected function create_tabs() {
 								'label' => esc_html__( 'Google Fonts Load', 'elementor' ),
 								'field_args' => [
 									'type' => 'select',
-									'std' => 'auto',
+									'std' => 'swap',
```
Comment (matipojo) [inline includes/settings/settings.php:341]: Revert

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request merged**